FreeRTOS/FreeRTOS
# Test: Mask unit test warning

Mask unit test warning

Description
-----------
Jenkins was failing the unit test because of a warning, removing the warning as it is previously tested

Test Steps
-----------
Unit Tests pass



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
